Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -168,30 +171,81 @@ namespace AZ
             return worklistData;
         }
 
-        constexpr size_t WorkListCapacity = 5;
-        using WorkListType = AZStd::fixed_vector<AzFramework::IVisibilityScene::NodeData, WorkListCapacity>;
+        // Used to accumulate NodeData into lists to be handed off to jobs for processing
+        struct WorkListType
+        {
+            AZStd::vector<AzFramework::IVisibilityScene::NodeData> m_nodes;
```

Comment:
Would a fixed_vector or an array be a better choice here?

Reply:
Size is controlled via a CVAR
Also is flexible depending on the job allocation strategy that is also controlled via CVAR